src/components/calcite-pick-list/calcite-pick-list.tsx

```diff
@@ -28,30 +28,29 @@ export class CalcitePickList {
 
   /**
    * When true, the items will be sortable via drag and drop.
-   * Only applies when mode is configuration
+   * This only applies when mode is configuration.
```

Comment:
This should be marked deprecated. Can you double check if you merged in latest from master?